### ISSUE TITLE
error when build tool with crystal 0.34.0

### ISSUE THREAD

**Issue by Pishoy** [threebotserver/publishingtools#149]
```
codescalerssMBP:publishingtools codescalers$ bash build.sh 
```
![image](https://user-images.githubusercontent.com/21987477/85050309-2dcf6a00-b196-11ea-95d0-7abf3e04587f.png)

```
codescalerssMBP:publishingtools codescalers$ crystal --version
Crystal 0.34.0 (2020-04-06)

LLVM: 6.0.1
Default target: x86_64-apple-macosx


```

**Comment by Pishoy** [comment]
```
    wget https://github.com/crystal-lang/crystal/releases/download/0.34.0/crystal-0.34.0-1.pkg -O crystal-0.34.0-1.pkg && \
    sudo installer -pkg crystal-0.34.0-1.pkg -target /Applications && rm crystal-0.34.0-1.pkg

```

````
codescalerssMBP:~ codescalers$ cd ~/code/publishingtools/
codescalerssMBP:publishingtools codescalers$ git log -1
commit f5f29ef9efdcf6be2142f4fdd7f03192d89d9483 (HEAD -> 0.34-version, origin/0.34-version)
Author: Ahmed T. Youssef <xmonader@gmail.com>
Date:   Tue Jun 23 12:47:34 2020 +0200

    update with a working version on 0.34 with crystaltools latest
~
```

**Comment by xmonader** [comment]
code updated to 0.35.1 and won't fix anything 0.34 related
